build.gradle

```diff
@@ -1635,8 +1627,14 @@ project(':streams:examples') {
   archivesBaseName = "kafka-streams-examples"
 
   dependencies {
+    // this dependency should be removed after we unify data API
+    compile(project(':connect:json')) {
```

Comment:
`compile` is deprecated, you have to choose between `api` and `implementation`.

Reply:
@ijuma well noted! I have updated the PR. Since there are many "compile" references in build.gradle, I will have a look later on JIRA if there is already a ticket to update all this to api or implementation and raise if not.

Reply:
having a better look, seems that no ticket is required. They are compileOnly or testCompileOnly which are not deprecated. All good.